src/index.html

```diff
@@ -7,12 +7,73 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="./styles/style.css"
     />
   </head>
+
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__item-img">
+        <a href="#APPLE_A1419_iMac_27">
+          <img
+            src="./images/imac.jpeg"
+            alt="APPLE_A1419_iMac_27"
+          />
+        </a>
+      </div>
+
+      <div class="card__item-content">
+        <h3 class="card__item-name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+
+        <div class="card__item-code">Product code: 195434</div>
+
+        <div class="card__item-review">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <div class="card__review">
+            <a href="#reviews">Reviews: 5</a>
+          </div>
+        </div>
+
+        <div class="card__item-price">
+          <p>Price:</p>
+          <p>$2,199</p>
+        </div>
+
+        <div class="card__btn-buy">
+          <a
+            href="#buy"
+            data-qa="hover"
+          >
+            Buy
+          </a>
+        </div>
```

Comment:
```suggestion
        <a
          class="card__btn-buy"
          href="#buy"
          data-qa="hover"
        >
          Buy
        </a>
```
you don't even need div wrapper. apply all styles directly to `a` (but using class selector) to make it look same as on the mockup